bindings/rust/s2n-tls-hyper/tests/common/certs/cert.pem

```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Yeah we can consolidate them! I'll do this in a separate PR since it's a bigger change: https://github.com/aws/s2n-tls/pull/4858

After they're moved to a central location I'll update this PR to re-use the same certs.